docs/troubleshooting-guide/index.md

```diff
@@ -33,6 +33,7 @@ Problems, their causes, how to detect them, and how to fix them.
 
 - [Saving questions or dashboards][proxies].
 - [My dashboard is slow][slow-dashboard].
+- [My SQL question (native query) doesn't work][sql].
```

Comment:
```suggestion
- [My SQL question doesn't work][sql].
```
Unless we're troubleshooting non-SQL questions as well, we can lose this qualifier.

Reply:
I originally thought to include it so that it's discoverable via search (on Discourse, I noticed that some people will refer to their problem using the term "native query" without mentioning "SQL" at all).

But you're right, "native query" should be used to distinguish from GUI. Also realised that "native query" would also include Mongo queries (and maybe other query languages?).

I'll keep a running list of these situations as I encounter them. It'll be interesting to pull a list of our glossary "aka"s too... maybe we can turn these things into tags at some point.🤞